developer_docs/dev-env.md

```diff
@@ -4,27 +4,29 @@ These docs help you get a full dev environment setup for working on this driver,
 
 ## Install Python
 
-You need Python3.6+ and pip. Install those according to the instructions for your operating system. 
+You need Python3.9+ and pip. Install those according to the instructions for your operating system. 
 
 For Ubuntu, you can do this:
 
 ```
 sudo apt update
 sudo apt install software-properties-common
 sudo add-apt-repository ppa:deadsnakes/ppa
-sudo apt install python3.7
-python3.7 --version
+sudo apt install python3.9
+sudo apt install python3.9-distutils
+sudo apt install libpython3.9-dev
+python3.9 --version
```

Comment:
are there more installs here because they removed some dependencies from the core python library?

Reply:
Sort of, I think we should have always included these in the instructions but didn't realise we needed them because I/others already had them. When I installed 3.9, I found that I discovered these were needed.